unit/analyses/ai/ai_simplify_lhs.cpp

```diff
@@ -33,6 +33,16 @@ class constant_simplification_mockt:public ai_domain_baset
   {}
   void make_entry() override
   {}
+  bool is_bottom() const override
+  {
+    UNREACHABLE;
+    return true;
```

Comment:
I think in general we do not need to return after `unreachable` since this is considered at least an exception if not an abort.

Reply:
True; although i don't see the harm in doing so.  It should avoid some over-zealous compiler warnings.